docs/strictdoc_01_user_guide.sdoc

```diff
@@ -1539,6 +1539,10 @@ The supported field types are:
    * - ``Reference``
      - **DEPRECATED:** comma-separated list with allowed reference types: ``ParentReqReference``, ``FileReference``. In the newer versions of StrictDoc (0.0.45+), a separate ``RELATIONS:`` section is used to configure the available relations.
 
+.. note::
+
+    The **Field Type** also influences how the field is presented in the web editor UI. A ``SingleChoice`` field enables an auto-completion function, which is useful in cases where the same values are entered repeatedly (such as requirement owners, test case types, or status fields).
```

Comment:
Here let's have it as `**Field type**`. I am going to explain this soon in the dev guide that we follow European capitalization for everything that is not a proper noun. (e.g. `Field type` but `Source View screen`).

Also, I see that the table header is also called `Field Type`. It is inconsistent and let's fix it here as well. Let's fix it to `Field type` as well.